### PR TITLE
Draft example of a JSON Schema concept catalog for biomedical studies

### PR DESCRIPTION
**Pull request by jfuerth**
Just putting down some concrete ideas for what a JSON Schema concept catalog could look like, to get the discussion going.

### CODE REVIEW

File: schemas/Observation.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          - 2000-04-15T14:23:06-04:00
+          - 2000-04-15T18:23:06+00:00
+      - format: date
+        examples:
+          - 2000-04-15
+  geo_location:
+    "$ref": https://schemablocks.org/schemas/ga4gh/v0.0.1/GeoLocation.json
+    description: |
+      The geographical location where the observation, assessment, or sample collection took place.
+  collected_by:
```
Comment (mcourtot) [inline schemas/Observation.yaml:41]: collected_by a Participant as below? I think you may want to have a Person, and then Subjects for the study - it seems the Participants are more like subjects based on the description, though labels are always hard. To me a PI in a study is a participant though not a subject for example.

**Review by mcourtot — COMMENTED** [review]
I really like the examples, thanks for putting this together. A concern in creating such a catalog was the search ability as the catalog grows. IIRC Bob Freimuth had specific suggestions on how to address this.

File: schemas/Observation.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    description: |
+      The role of the person who made the observation or collected the sample.
+    oneOf:
+      - "$ref": https://schemablocks.org/schemas/sb-phenopackets/OntologyClass/v1.0.0
+      - type: string
+        description: Free text description of the role
+    examples:
+      - { "id": "NCIT:C51863", "label": "Research Clinician" }
```
Comment (mbaudis) [inline schemas/Observation.yaml:49]: "Yamlify" the examples?
```
      - id: "NCIT:C51863"
        label: "Research Clinician"
      - id: "NCIT: C51864"
        label: "Research Nurse"

File: schemas/Observation.yaml (hunk 1 above)
Comment (jfuerth) [reply schemas/Observation.yaml:41]: Great points. That all makes sense to me.

File: schemas/Observation.yaml (hunk 2 above)
Comment (jfuerth) [reply schemas/Observation.yaml:49]: Sure, I can do that. Wasn't sure of the best tradeoff between vertical compactness and YAML purity. :)

Comment (mbaudis) [reply schemas/Observation.yaml:49]: I have no clue myself :-)

File: schemas/Observation.yaml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    type: string
+    oneOf:
+      - format: date-time
+        examples:
+          - 2000-04-15T14:23:06-04:00
+          - 2000-04-15T18:23:06+00:00
+      - format: date
+        examples:
+          - 2000-04-15
+  geo_location:
```
Comment (mbaudis) [inline schemas/Observation.yaml:37]: The `geo_location` object has always been a bit of a draft. We've now changed to a version, not yet propagated to {S}[B]: https://progenetix.org/services/schemas/GeoLocation/ ... which uses clean GeoJSON & overloads it with some annotations in `properties` (yes, that's the way geoJSON does it for annotations ...). We'll update this one here, too.

**Comment by mbaudis** [comment]
I'll merge this - is for discussion anyway with "playground" status; can be amended in master.

**Pull request merged**